### DISCUSSION TITLE
[Oracle]  Pythonic way to call Oracle PL/SQL

### DISCUSSION THREAD

**Discussion by jiaulislam** [sqlalchemy/sqlalchemy#8420] [Usage Questions] [answered]
okay so i had to go back to raw oracledb to call oracle stored procedures or functions that returns cursor in my app. But i really wanted to use sqlalchemy to do that but couldn’t find suitable python way. Is there is any support plan for that or any suggestions how to achieve  that with sqlalchemy?

**Comment by zzzeek** [comment, marked as answer]
for calling stored procedures and other special DBAPI-level functions you can access the cursor from SQLAlchemy following the steps at https://docs.sqlalchemy.org/en/14/core/connections.html#working-with-the-dbapi-cursor-directly
